api/server/controllers/auth/login.controller.js

```diff
@@ -0,0 +1,39 @@
+const User = require('../../../models/User');
+
+const loginController = async (req, res) => {
+  try {
+    const user = await User.findById(
+      req.user._id
+    );
+
+    // If user doesn't exist, return error
+    if (!user) { // typeof user !== User) { // this doesn't seem to resolve the User type ??
```

Comment:
note: I had to change this line and I'm not really sure why the type changed but I also don't see how you could get to this line of code without it being the User object either. Can't see any problems